include/mongoose.hrl

```diff
@@ -34,3 +34,11 @@
 %% ---------------------------------
 %% Logging mechanism
 -include("mongoose_logger.hrl").
+
+%% for worker_pool
+-record(mongoose_worker_pool, {name :: atom(),
```

Comment:
This record is far better then then prev one. I'd like to know why you didn't decide to use map instead of the record as I suggested.

Reply:
Personally I don't have a strong opinion here. Such a simple record will be easy to read in console but will still require consulting with header file (if someone e.g. forgets the order of timeouts). On the other hand - such a simple map will be difficult to abuse and easy to spec. I'm slightly leaning towards the latter option but I won't complain about a record for now.